tools/process-bounties/indexer.js

```diff
@@ -35,7 +35,7 @@ bounties.withPromise().then(async bountyPaths => {
             { Bounty: { Credit: parseInt(vulnerabilityDetails.CVSS.Score * 100), Cash: 25 } }
 
         bountiesToIndex.push({
-            "ID": `${vulnerabilityDetails.PackageVulnerabilityID}-${vulnerabilityDetails.Package.Registry}-${encodeURIComponent(vulnerabilityDetails.Package.Name)}`,
+            "ID": `${vulnerabilityDetails.PackageVulnerabilityID}-${vulnerabilityDetails.Package.Registry}-${vulnerabilityDetails.Package.Name.replace('/','-')}`,
```

Comment:
Revert this change for now, and we will track as an alternative PR.

Reply:
Lol, was also just reviewing - will do.

Reply:
Cheers! 🍰